spec/defines/resource_server_spec.rb

```diff
@@ -904,18 +904,6 @@
         it { is_expected.to contain_nginx__resource__location("#{title}-default").with_ssl_only(true) }
       end
 
-      context 'SSL cert missing' do
```

Comment:
the datatypes enforce the correct param, so we don't need to test wrong/missing parameter. We dropped these kinds of tests on all other puppet4 only modules as well.